README.md

```diff
@@ -18,14 +18,24 @@ pip install opencv-python
 
 ## Get the data
 
-Go to the [dataset presentation page](https://www.prophesee.ai/2019/12/18/atis-automotive-detection-dataset/) and download the dataset (200G compressed and 750G uncompressed !).
+### 1 Megapixel Automotive Detection Dataset
 
-The dataset is split into 10 archive files that can be independently used (2 for testing and validation sets each and six for training set)
+Go to the [dataset presentation page](https://www.prophesee.ai/2020/11/24/automotive-megapixel-event-based-dataset/) and download the dataset.
+The dataset is split between train, test and val folders. 
+Files consist of 60 seconds recordings that were cut from longer recording sessions. Cuts from a single recording session are all in the same training split.
+
+### GEN1 Automotive Detection Dataset 
+
+Go to the [dataset presentation page](https://www.prophesee.ai/2019/12/18/atis-automotive-detection-dataset/) and download the dataset.
+(200G compressed and 750G uncompressed !). 
+The dataset is split into 10 archive files that can be independently used (2 for testing and validation sets each and six for training set).
 Each archive contains up to 500 files and their annotations.
 
-Unzip using [7zip](https://www.7-zip.org/).
+### GEN1 N-CARS Dataset
+
+Go to the [dataset presentation page](https://www.prophesee.ai/2018/03/13/dataset-n-cars/) and download the dataset.
+The dataset is split in 7940 car and 7482 background training samples, 4396 car and 4211 background testing samples. Each example lasts 100 milliseconds.
 
-If you use the dataset, please cite the article ["A Large Scale Event-based Detection Dataset for Automotive" by P. de Tournemire, D. Nitti, E. Perot, D. Migliore and A. Sironi](https://arxiv.org/abs/2001.08499)
```

Comment:
Can you add a similar sentence, but for the NeurIPS paper?

Reply:
Those sentences are already in the HTML pages of our corporate web site where users can download the datasets AND  also in the README of the Datasets. I was not sure this README was the right place to add it a third time...

If I do add the "please cite the article", then I should do it for the 3 datasets, no?

Reply:
I see, the fact is that someone could use the evaluation code without any of our datasets. 
The citation to the neurips paper will cover the case in which people are just using the evaluation code.